staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5046,6 +5046,10 @@ const (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // Event is a report of an event somewhere in the cluster.
+//
+// There is a pending enhancement to replace `Event.v1.` with what is currently
+// `Event.v1beta1.events.k8s.io`; see <https://github.com/kubernetes/enhancements/issues/383> and
+// <https://git.k8s.io/community/contributors/design-proposals/instrumentation/events-redesign.md>.
```

Comment:
We have to migrate the proposal to KEP.
@yastij - do you have time to do that? If not I can do that next weekend when i will be flying for Kubecon.

Reply:
sgtm, I'll take look once your PR is out